docker-orchestration/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9-slim-buster
+
+RUN apt-get update \
+    && apt-get -q -y install git gcc nginx python3-libvirt libxslt1-dev libxml2-dev qemu-kvm libvirt-dev virt-manager mediainfo
+
+WORKDIR /var/www
+RUN git clone https://github.com/CCExtractor/sample-platform.git
+
+WORKDIR /var/www/sample-platform/
+RUN python -m pip install --upgrade pip setuptools wheel \
+    && pip install -r requirements.txt
+
+RUN cp "install/platform" /etc/init.d/platform \
```

Comment:
It seems to me that we could have the command that's in the platform startup script as the container run script, and configure the container to restart as soon as it dies. That's a more "docker"-like way than using an initscript that I'm not sure will be running by itself if you restart the container.

Reply:
fixed via 32fb56d